release-notes.md

```diff
@@ -0,0 +1,60 @@
+# Release v0.9.0
+
+## New Features
+* Add pai service dashboard to grafana. - [PR 1694](https://github.com/Microsoft/pai/pull/1694)
```

Comment:
@xudifsd  may we have an introduction for the PAI Service Dashboard, there is only one picture in PR 1694 now.

Reply:
Maybe "Cluster admin can now view resource consumption by pai services in PaiServiceMetrics page"?